pubpub/pubpub
# Bh/spubheader restructure

Implements #1853

+ Consolidates and narrows CSS
+ Performs preliminary wire-up of #1852, to be finalized by that PR's merge

## _Test plan_
+ open an incomplete spub
+ confirm look + behavior of `SpubHeaderToolbar` at top
+ click `Begin Submission` at bottom of `SpubHeader`, it should take you to the `Submission` tab
